### PR TITLE
Slack message cleanup

### PR DESCRIPTION
**Pull request by aanari**
Ready for review!

### CODE REVIEW

File: lib/github_event_handler.rb [hunk 1]
```diff
@@ -13,6 +13,14 @@ def organization
...
     @data["name"]
   end
+
+  def to_s
+    to_slack_string
+  end
+
+  def to_slack_string
+    "<http://github.com/#{@data["full_name"]}|#{@data["name"]}>"
+  end
```
Comment (aanari) [inline lib/github_event_handler.rb]: Link directly to Repo link, display short name

File: lib/github_event_handler.rb [hunk 2]
```diff
@@ -55,7 +63,7 @@ def to_s
   end
 
   def to_slack_string
-    "@#{login}"
+    "<http://github.com/#{login}|#{login}>"
```
Comment (aanari) [inline lib/github_event_handler.rb]: Link directly to GH user profile

File: server.rb [hunk 3]
```diff
@@ -23,21 +18,15 @@ def review_label!(repository, issue)
...
+  elsif comment.matches?(/\+1/i)
+    message = "Thumbs up :+1:"
   elsif comment.matches?(/lgtm/i)
-    if issue.owner?(comment.user)
-      message = "thinks their issue is great"
-    else
-      message = "has a positive opinion about this"
-    end
+    message = "Looks good :check:"
   end
```
Comment (aanari) [inline server.rb]: Shorten these and use icons

File: server.rb [hunk 4]
```diff
@@ -23,21 +18,15 @@ def review_label!(repository, issue)
...
 
   if message
-    return "#{team_name_for_project(repository)}#{issue} #{comment.user} #{message}\n```#{comment.body.slice(0...100)}```"
+    return "> #{repository} #{issue} #{comment.user}: #{message}\n>>>#{comment.body.slice(0...100)}..."
```
Comment (aanari) [inline server.rb]: Display content in quoted blocks instead of backtick blocks to allow hilighted nicks and emojis to come through

**Comment by cohitre** [comment]
lgtm

**Comment by aanari** [comment]
+1

**Comment by cohitre** [comment]
lgtm

**Comment by cohitre** [comment]
ping

**Comment by aanari** [comment]
ping

**Comment by aanari** [comment]
pong

**Comment by aanari** [comment]
ping

**Comment by cohitre** [comment]
ping

**Comment by aanari** [comment]
@api

**Comment by aanari** [comment]
ping :tiger: @api

**Comment by aanari** [comment]
LGTM :100:

**Comment by aanari** [comment]
testing

**Comment by aanari** [comment]
something

**Comment by aanari** [comment]
pang

**Pull request merged**